transfermonitor/transfermonitor.go

```diff
@@ -28,6 +30,7 @@ import (
 const (
 	maxBlocksProcessedPerIteration = 100000
 	destinationChainID             = "osmosis-1"
+	orderSubmittedEventSignature   = "0x59f858504f8d8ad967dd7453df850e265270474e364b7e2fbd3333e06efdbfc0"
```

Comment:
@thal0x can you please confirm this doesn't change? i wasnt sure if there's a better way to get the events we neeed